Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,79 @@
+description: 'Unshorten urls through a tor proxy server. Unshorten urls onsite with
+  the power of Tor to prevent leaking ip addresses to adversaries.  Created by CypherInt.  Visit
+  us at https://cypherint.com.  Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/'
+detaileddescription: cloaken will return the unshortened version of a url
+configuration:
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Server URL
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |-
+    # The command demisto.command() holds the command sent from the user.
+    from cloakensdk.client import SyncClient
+    from cloakensdk.resources import Url
+
+    def get_client():
+        server=demisto.params()["server_url"]
+        password=demisto.params()["credentials"]["password"]
+        username=demisto.params()["credentials"]["identifier"]
+        client=SyncClient(server_url=server,
```

Comment:
Can we pass `verify` here?

Reply:
The setup script for AWS requires a signed certificate through letsencrypt, but if people are putting this behind a LAN it may not validate.  I can add this feature.

Reply:
I added the verify parameter